scripts/docbuild.rb

```diff
@@ -30,5 +30,6 @@ def execute(cmd)
 
 $log.info("Copying docs to fileshare...")
 FileUtils.rm_rf DOCS_FILESHARE
+FileUtils.mkdir_p DOCS_FILESHARE
```

Comment:
wasn't sure if it was an asciibinder bug, all works well if you browse the built stuff under _package/community/, it has the master/ subdirectory there.